tests/LineBot.Tests/RichMenuRequestTests/TheRichMenuBoundsClassTests.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright 2017-2018 Dirk Lemstra (https://github.com/dlemstra/line-bot-sdk-dotnet)
```

Comment:
This file should be move to `tests/LineBot.Tests/RichMenu/RichMenuBoundsTests/` and split into a separate file for each property.